src/pudl/etl/__init__.py

```diff
@@ -67,10 +69,10 @@
     *load_assets_from_modules([pudl.analysis.mcoe], group_name="mcoe"),
     *load_assets_from_modules([pudl.output.ferc1], group_name="out_ferc1"),
     *load_assets_from_modules(
-        [pudl.analysis.service_territory], group_name="service_territory_eia861"
+        [pudl.analysis.service_territory], group_name="out_service_territory_eia861"
     ),
     *load_assets_from_modules(
-        [pudl.analysis.state_demand], group_name="state_demand_ferc714"
```

Comment:
There was some chatter last round about whether this should be `ferc714` or nod to the inclusion of EIA 861 data in these outputs as well, but I think leaving it this way is ok.

Reply:
Ah that's right. I'm down to keep it 714 for now given the majority of the data is from ferc714.